### PR TITLE
BAE Systems 

### PR DESCRIPTION
**Pull request by crablab**
Adds details for the BAE systems event. 

Current awaiting details from them and background image :+1:

### CODE REVIEW

**Comment by crablab** [comment]
Updated with event details - ready to go :raised_hands:

File: _events/baectf.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+name: "Capture the Flag with BAE Systems"
+is-public: true
+is-over: false
+
+start: "2019-01-19T10:00:00+00:00"
+end: "2019-01-19T19:16:00+00:00"
+
+short-description: ""
+full-description: "Local Hack Day is a global hackathon uniting thousands of students. Participants across 100+ distributed campuses are able to connect with their local communities and other hackers around the world through digital workshops & mini-events, social media, and a global livestream."
```
Comment (Alex-CD) [inline _events/baectf.md]: Did you mean to leave the local hack day description? Also, this needs a short description if possible :)

File: _events/baectf.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+# Used for styling
+color: "#DA1E1E"
+display-date: "19th Jan 2019"
+layout: event
+location: "Windsor Building"
+jumbotron-image: localhost-2.jpg
```
Comment (Alex-CD) [inline _events/baectf.md]: localhost-2 doesn't exist(?).

**Comment by crablab** [comment]
Not entirely sure what happened as the commit with all the changes in was already added to the PR :thinking:

**Comment by crablab** [comment]
@Alex-CD ping

**Comment by Alex-CD** [comment]
Pong.

**Pull request merged**